test-packages/ember-cli-fastboot-testing-app/package.json

```diff
@@ -0,0 +1,68 @@
+{
+  "name": "ember-cli-fastboot-testing-app",
+  "version": "0.0.0",
+  "private": true,
+  "description": "Small description for ember-cli-fastboot-testing-app goes here",
+  "repository": "",
+  "license": "MIT",
+  "author": "",
+  "directories": {
+    "doc": "doc",
+    "test": "tests"
+  },
+  "scripts": {
+    "build": "ember build",
+    "build:prod": "ember build --environment=production",
+    "lint": "npm-run-all --aggregate-output --continue-on-error --parallel lint:*",
+    "lint:hbs": "ember-template-lint .",
+    "lint:js": "eslint .",
+    "start": "ember serve",
+    "test": "npm-run-all lint:* test:*",
+    "test:ember": "ember test"
+  },
+  "devDependencies": {
+    "@ember/optional-features": "^1.3.0",
+    "@glimmer/component": "^1.0.0",
+    "@glimmer/tracking": "^1.0.0",
+    "babel-eslint": "^10.1.0",
+    "broccoli-asset-rev": "^3.0.0",
+    "chai": "^4.2.0",
+    "chai-fs": "^2.0.0",
+    "chai-string": "^1.5.0",
+    "ember-auto-import": "^1.5.3",
+    "ember-cli": "~3.19.0",
+    "ember-cli-app-version": "^3.2.0",
+    "ember-cli-babel": "^7.20.5",
+    "ember-cli-dependency-checker": "^3.2.0",
+    "ember-cli-fastboot": "3.2.0-beta.2",
+    "ember-cli-fastboot-testing": "^0.5.0",
+    "ember-cli-htmlbars": "^5.1.2",
+    "ember-cli-inject-live-reload": "^2.0.2",
+    "ember-cli-sri": "^2.1.1",
+    "ember-cli-uglify": "^3.0.0",
+    "ember-data": "~3.19.0",
+    "ember-export-application-global": "^2.0.1",
+    "ember-fetch": "^8.0.1",
+    "ember-load-initializers": "^2.1.1",
+    "ember-maybe-import-regenerator": "^0.1.6",
+    "ember-qunit": "^4.6.0",
+    "ember-resolver": "^8.0.0",
+    "ember-source": "~3.19.0",
+    "ember-template-lint": "^2.8.0",
+    "ember-welcome-page": "^4.0.0",
+    "eslint": "^7.1.0",
+    "eslint-plugin-ember": "^8.6.0",
+    "eslint-plugin-node": "^11.1.0",
+    "execa": "^5.0.0",
+    "fs-extra": "^10.0.0",
+    "loader.js": "^4.7.0",
+    "npm-run-all": "^4.1.5",
+    "qunit-dom": "^1.2.0"
+  },
+  "engines": {
+    "node": "10.* || >= 12"
+  },
+  "ember": {
+    "edition": "octane"
+  }
+}
```

Comment:
```
  "volta": {    "extends": "../../package.json"  },
```

volta config.  

Also, I think we could strip down the devDeps to only necessary packages for your tests

Reply:
Included this. Thanks for the suggestion 👍